packages/sdk-cli/src/commands/screenshot.test.ts

```diff
@@ -10,6 +11,10 @@ import commandTestHarness from '../testUtils/commandTestHarness';
 import { homedir } from 'os';
 
 jest.mock('../models/captureScreenshot');
+jest.mock('open', () => ({
+  __esModule: true,
```

Comment:
Never seen this done before in a mock, why is it required?

Why does the `jest.mock` auto-mock not work?

Reply:
Thanks!
Not sure why I wrote it that way